content/6_3_simulation-general.html

```diff
@@ -490,7 +490,7 @@ <h5>Review Tips</h5>
                                 </div>
                                 <div class="row">
                                     <div class="col-12 border-bottom border-left border-right">
-                                        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#imageModal" data-full="../assets/images/general-table-in-input-verification-and-results-summary-report.png">
```

Comment:
missing `/` before assets causes broken image for Design Builder's Annotated Report